### PR TITLE
Displays game with a few functions working

### PR DESCRIPTION
**Pull request by JamesBuabin14**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Project Demo</title>
+    <script src="./js/jquery-3.3.1.min.js"></script>
```
Comment (JSChilds) [inline index.html]: This file has not been pushed up to the repository, therefore jQuery is not linked and none of your JavaScript works.

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            array[1].style.display = "none";
+            array = [];
+          }
+
+      })
+
+
+    }
+
+
```
Comment (JSChilds) [inline js/app.js]: Don't leave in big, inconsistent spaces like this, it makes the code hard to read.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Get your jQuery linked up and fix your spacing to make your code more readable.

File: js/app.js [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+  document.getElementById("score").innerHTML = " Your score :" + score; //Gets a score that has been counted up
+
+  for (var i = 0; i < cards.length; i++) { //loop which goes through the card array
+    cards[i].setAttribute("id", card_array[i]); //sets each with an with an ID from the card_array
+    cards.className += (" selected"); //confirms if the card has been selected
+
+
+
+    cards[i].addEventListener('click', function(e) { //On the event of a click on a card
+      // console.log(e.target);
```
Comment (JSChilds) [inline js/app.js]: Please remove ALL commented-out code and console.logs

File: pages/game.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  <meta charset="utf-8">
+  <title>Project Demo</title>
+  <script src="../js/jquery-3.3.1.min.js"></script>
+  <script src="../js/app.js" charset="utf-8"></script>
+  <link rel="stylesheet" type="text/css" href="../css/style.css">
+</head>
+
+<body>
+  <div class="stats-container">
+    <span id="timer"></span>
```
Comment (JSChilds) [inline pages/game.html:14]: Please change the font of these so they're easier to see.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Please make changes requested in comments.

**Comment by JamesBuabin14** [comment]
Changes made, waiting to make new pull request to master branch

**Review by JSChilds — APPROVED** [review]

**Pull request merged**